_tools/list-unused-images.sh

```diff
@@ -9,19 +12,17 @@ rm -f tmp-unused-images-history
 # Exceptions are ignored, and for .svg files we also look for potential .png
 # files with the same base name, as they might be sources.
 
-exceptions="docs_logo.svg tween_cheatsheet.png"
-
-files=$(find -name "_build" -prune -o \( -name "*.png" -o -name "*.jpg" -o -name "*.svg" -o -name "*.gif" \) -print | sort)
+files=$(find -name "_build" -prune -o \( -name "*.png" -o -name "*.webp" -o -name "*.jpg" -o -name "*.svg" -o -name "*.gif" \) -print | sort)
```

Comment:
I guess in theory, this should/could use `-iname` to be case insensitive (to match things like `FILE.WEBP` or `file.WebP`), or use `*.[Ww][Ee][Bb][Pp]` if we don't want to depend on GNU `find`, as `-iname` seems GNU-specific.